opencensus/stats/view_data.h

```diff
@@ -69,7 +69,13 @@ class ViewData {
   const DataMap<int64_t>& int_data() const;
   const DataMap<Distribution>& distribution_data() const;
 
+  // DEPRECATED: Returns a start time for the view data.
   absl::Time start_time() const;
+
+  // A map from tag values (corresponding to the keys in the ViewDescriptor, in
+  // that order) to the start time for those tags.
```

Comment:
It might be worth expanding this: "i.e. the first time a point was seen with that combination of tag values."

Reply:
Done.